src/Symfony/Component/Intl/Resources/emoji/composer.lock

```diff
@@ -0,0 +1,323 @@
+{
+    "_readme": [
+        "This file locks the dependencies of your project to a known state",
```

Comment:
what about not committing this file? I don't think it's useful to have it in the repo